src/transformers/__init__.py

```diff
@@ -3944,6 +3971,7 @@
         from .image_utils import ImageFeatureExtractionMixin
         from .models.beit import BeitFeatureExtractor, BeitImageProcessor
         from .models.bit import BitImageProcessor
+        from .models.blip import BlipImageProcessor, BlipProcessor
```

Comment:
BlipProcessor shouldn't be here

Reply:
I see, then it should be also removed for `Flava` as well no?

Reply:
In any case, removed it in f5978b7